sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -287,6 +310,28 @@ def test_scikit_vs_scipy():
     assert_raises(ValueError, _hc_cut, n_leaves + 1, children, n_leaves)
 
 
+def test_identical_points():
```

Comment:
Wouldn't hurt to run this test on all linkage types

Reply:
Done.